raphtory-graphql/src/model/graph/graph.rs

```diff
@@ -135,16 +135,122 @@ impl GqlGraph {
             .collect()
     }
 
-    async fn edges<'a>(&self, filter: Option<EdgeFilter>) -> Vec<Edge> {
-        match filter {
-            Some(filter) => self
-                .graph
+    async fn search_count(&self, query: String) -> usize {
```

Comment:
Tantivy has a collector https://docs.rs/tantivy/latest/tantivy/collector/struct.Count.html which allows us to do this a bit better - will need to push it internally